src/zepben/evolve/streaming/grpc/connect.py

```diff
@@ -7,374 +7,168 @@
-                        client_secret: Optional[str] = None,
-                        username: Optional[str] = None,
-                        password: Optional[str] = None,
-                        pkey=None,
-                        cert=None,
-                        ca=None,
-                        token_fetcher: Optional[ZepbenTokenFetcher] = None,
-                        secure=False,
-                        verify_auth_certificates=False,
-                        conf_ca=None,
-                        auth_ca=None):
-    """
-    Connect to a Zepben gRPC service.
-
-    `host` The host to connect to.
-    `rpc_port` The gRPC port for host.
-
-    `conf_address` The complete address for the auth configuration endpoint. This is used when an `token_fetcher` is not provided.
-        Defaults to http://<host>/auth
-    `secure` True if SSL is required, False otherwise (default). Must be True for authentication settings to be utilised.
-
-    One of the following sets of arguments must be provided when authentication is configured on the server:
-        `client_id` and `client_secret` for client credentials based authentication (usually M2M tokens), or
-        `client_id`, `username`, and `password` for user credentials authentication. Note this is not recommended for untrusted and insecure servers as
-        user credentials will be sent to the server in the clear. Or,
-        `authenticator`, for custom connection options.
-
-    `client_id` Your client id for your OAuth Auth provider.
-    `client_secret` Corresponding client secret if required.
-
-    `username` The username to use for an OAuth password grant.
-    `password` Corresponding password. If both `username` and `password` are provided, it takes precedence over the above client credentials.
-
-    `token_fetcher` A token fetcher that can provide OAuth2 tokens the `host` can validate. If this is provided, it takes precedence over the above credentials.
-
-    `pkey` Private key for client authentication if client authentication is required.
-    `cert` Corresponding signed certificate if client authentication is required. CN must reflect your hosts FQDN, and must be signed by the servers CA.
-    `ca` CA trust for the server, or None to use OS default certificate bundle.
-
-    `verify_auth_certificate` Whether to authenticate the certificate from the authentication provider.
-    `auth_ca` Filename of CA bundle for the authentication provider, or None to use OS default certificate bundle.
-
-    Raises `ConnectionError` if unable to make a connection to the server.
-    Returns a gRPC channel
-    """
-    warnings.warn('The connect_async function is deprecated. It will be replaced with other functions like connect_secure_async.',
-                  DeprecationWarning, stacklevel=2)
-    yield _conn(host, rpc_port, conf_address, client_id, username, password, client_secret, pkey, cert, ca, token_fetcher, secure, verify_auth_certificates,
-                conf_ca, auth_ca)
```

Comment:
It's about time we remove these in favour of the new `connect_*` functions. Let me know if you have any objections

Reply:
Yep happy for them to be removed.